js/src/provider/WalletLinkProvider.ts

```diff
@@ -4,23 +4,34 @@
 
 import SafeEventEmitter from "@metamask/safe-event-emitter"
 import BN from "bn.js"
-import { ethErrors, serializeError } from "eth-rpc-errors"
+import { ethErrors } from "eth-rpc-errors"
 
 import { WalletLinkAnalytics } from "../connection/WalletLinkAnalytics"
-import { EthereumChain } from '../EthereumChain'
+import { EthereumChain } from "../EthereumChain"
```

Comment:
Theres a lot of prettier/lint changes that didn't make it in last time. I commented where the actual changes in this file are.

Reply:
Uri44derr

Reply:
Thank you
![fb bin (1)](https://user-images.githubusercontent.com/96317554/146823805-ff1ef0e9-1d20-45ba-a09d-2a2b4a6e0763.png)